bundles/framework/tools.vitruv.framework.vsum/src/tools/vitruv/framework/vsum/repositories/ResourceRepositoryImpl.java

```diff
@@ -398,7 +400,7 @@ public Void call() throws Exception {
                     resource.delete(null);
                     ResourceRepositoryImpl.this.modelInstances.remove(vuri);
                 } catch (final IOException e) {
-                    logger.info("Deletion of resource " + resource + " did not work. Reason: " + e);
+                    logger.error("Deletion of resource " + resource + " did not work. Reason: " + e);
```

Comment:
Preferably:

```suggestion
                    logger.error("Deletion of resource " + resource + " did not work.", e);
```

to give the logging framework control over formatting the exception.